docs/apis-tools/administration-sm-api/sidebar-schema.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     "apis-tools/administration-sm-api/administration-sm-api-overview",
     "apis-tools/administration-sm-api/administration-sm-api-authentication",
     {
-      Specifications: require("./specifications/sidebar.js"),
+      Specifications: require("./specifications/sidebar"),
```

Comment:
These sidebar paths all change, because the generator now generates .ts files instead of .js.